app/models/host/base.rb

```diff
@@ -74,6 +74,9 @@ def self.taxonomy_conditions
     scope :no_location,     -> { rewhere(:location_id => nil) }
     scope :no_organization, -> { rewhere(:organization_id => nil) }
 
+    # If skip_orchestration is set to true, all orchestration calls will be skipped
+    attr_accessor :skip_orchestration
```

Comment:
I would rather call this 'importing_facts' or similar. It's really what it is isn't it? `:someorchestrationtask, :unless => :importing_facts` is more clear about what's going on too IMO. 

We really should move the facts importing logic to a concern :smile: